ads/pixels.md

```diff
@@ -0,0 +1,64 @@
+<!---
```

Comment:
Please link to this doc from [here](https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad/amp-ad.md#supported-ad-networks)

Reply:
Linked.